### PR TITLE
create-cloudflare: replace create-svelte with sv

### PR DESCRIPTION
**Pull request by codenoid**
## What this PR solves / how to test

create-svelte is soon to be deprecated

[Fixes #[insert GH or internal issue number(s)].](https://github.com/sveltejs/kit/pull/12841#issuecomment-2425258043)

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Tests included
  - [x] Tests not necessary because: existing test is compatible
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required
  - [x] Not required because: required, passing here: #7088
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Changeset included
  - [ ] Changeset not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Documentation not necessary because: no related existing documentation on the replaced package

### CODE REVIEW

**Comment by codenoid** [comment]
tested on my local, please review

**Comment by emily-shen** [comment]
hey thanks for opening this PR :) 
I'm running our e2e's for you here, since they can't be run from a fork: https://github.com/cloudflare/workers-sdk/pull/7088 

I think they're probably going to need updating - since running e2e's locally is a faff, I'm happy to make those changes and push to this branch if that's okay with you?

Once e2e's are sorted, this looks good to me - thanks again!

**Comment by codenoid** [comment]
yes please @emily-shen

**Comment by codenoid** [comment]
image

svelte e2e is now passed

**Review by emily-shen — APPROVED** [review]

File: .changeset/hip-actors-thank.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"create-cloudflare": minor
+---
+
+create-svelte is soon be deprecated, use the new official replacement npx sv instead.
```
Comment (andreaselia) [inline .changeset/hip-actors-thank.md:5]:
```suggestion
create-svelte is soon to be deprecated; use the new official replacement npx sv instead.
```

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts [hunk 2]
```diff
@@ -357,15 +357,20 @@ function getFrameworkTests(opts: {
 			svelte: {
 				promptHandlers: [
 					{
-						matcher: /Which Svelte app template/,
+						matcher: /Which template would you like/,
```
Comment (andreaselia) [inline packages/create-cloudflare/e2e-tests/frameworks.test.ts:360]: Why this change?

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts [hunk 3]
```diff
@@ -357,15 +357,20 @@ function getFrameworkTests(opts: {
...
 						input: [keys.enter],
 					},
 					{
-						matcher: /Add type checking with TypeScript/,
+						matcher: /Add type checking with Typescript/,
```
Comment (andreaselia) [inline packages/create-cloudflare/e2e-tests/frameworks.test.ts:364]: It should be "TypeScript" not "Typescript", please revert.

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts (hunk 2 above)
Comment (emily-shen) [reply packages/create-cloudflare/e2e-tests/frameworks.test.ts:360]: These changes are to do with the sv package, not create-cloudflare.

File: packages/create-cloudflare/e2e-tests/frameworks.test.ts (hunk 3 above)
Comment (emily-shen) [reply packages/create-cloudflare/e2e-tests/frameworks.test.ts:364]: This is from sv not us

**Comment by emily-shen** [comment]
For posterity, c3 e2e's were passing with the linked branch #7088, didn't realise closing that PR would remove the actions whoops 😅
